### ISSUE TITLE
🐛 Do not disconnect the bot

### ISSUE THREAD

**Issue by rockrevenchy** [eritislami/evobot#1518]
**Describe the bug**
One of my friends has this nasty habit to disconnect the bot (via the discord rightclick menu in the voicechat)

The bot just glitches and seems to hang, it wont reconnect and it is not possible to stop it either, only fix is to stop the instance and restart it

**How To Reproduce**
1. Have the bot play a song in a voicechat
2. Disconnect it by force if you have the required permissions
 
**Expected behavior**
The bot should realize it has been disconnected and act accordingly (stop current playlist and wait to be called back)

**Environment (add if possible)**
* Node.js version: v16.20.0

**Comment by LucaBarbaLata** [comment]
Just restart the bot if u neef it asap. Ig they will fix it. I also had this problem and this is what ive done. Restart ur raspi

**Comment by rockrevenchy** [comment]
yeah this is my current fix but I though id report this issue still

**Comment by rockrevenchy** [comment]
Well it prob shouldn't be automatically closed since it is still an ongoing issue
